documentation/MeetingMinutes/Week50_syncCalls.md

```diff
@@ -0,0 +1,99 @@
+# Week 50 meeting
+
+## Participants
+
+13 December 2022:
+|No|Name|Company|
+|:---:|:---|:---:|
+|1|Dawid Wróblewski **(organizer)**|DT|
+|2|Jens Pöschl|DT|
+|3|Andrzej Ochocki|DT|
+|4|Ludovic ROBERT|Orange|
+|5|Patrice CONIL|Orange|
+|6|MARIA VIRTUDES BRIZ RUIZ|Telefonica|
+|7|MONA MOKHBER|Telefonica|
+|8|José Luis Urien Pinedo|Telefonica|
+|9|GLYN POVAH|Telefonica|
+|10|Adri Loloci|Vodafone|
+|11|Mus Dumenci|Hutchison|
+|12|Fabrizio Moggio|TIM Italy|
+|13|Helene Vigue|GSMA|
+|14|Toyeeb Rehman|GSMA|
+|15|Mark Cornall|GSMA|
+
+17 November 2022:
+|No|Name|Company|
+|:---:|:---|:---:|
+|1|Maria Virtudes Briz Ruiz|Telefonica|
+|2|Jung Yun-sun (Serena)|SK Telecom|
+|3|Toshi Wakayama|KDDI|
+|4|Harin Honesyandi Parandika|XL Indonesia|
+|5|Heny Setywan|XL Indonesia|
+|6|Dawid Wróblewski|DT|
+
+## Agenda
+
+1. Current activities status & introduction to guests
+2. presenting Mobile Identity APIs
+3. Telefonica presentation (10min)
+4. Vodafone presentation (10min)
+5. DT presentation (10min)
+6. Q&A session
+
+## MoM
+
+1. Dawid Wroblewski (DW) presented the status of current activities inside GitHub:
+### **simSwap API**
+  #### Recommendation:
+  >Enable simSwap API via OAUTH2.0 flows:
+  - OAUTH2.0 code flow/OpenID Connect
+  - OAUTH2.0 Client Credentials grant
+  
+  >Enable simSwap API flavors:
+  - based on dedicated endpoint (Orange + TEF proposal)
+  - based on MC ATP API description
+
+MNOs continue work and aiming to deliver APIs by the end of this week.
+
+   -------
+
+   >following items were not presented during meeting, although sharing status and recommendations
```

Comment:
I think this part still might be misleading because you might have mentioned some of these points in the meeting but the rest of discussion and the answeres to your suggestion are missing here. 
For example here it says that in the other API defenition "IP Address+Port passed in resource call" and it's not true. IP address is and was not included in any resource call. It uses 3-legged token and  requires authentication via **mobile network** (excluding for example by SMS/OTP or user/password as an authentication method). It's NOT a query API either. 
If someone reads this MoM might think that it was discussed and agreed in the meeting that the other proposal is a bad idea and it's not valid when it's not what happened. Could you please edit it?